### PR TITLE
YieldOne Bid Adapter: add Flux Wrapper support.

### PR DESCRIPTION
**Pull request by kyoya-takei**
## Type of change
- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
- Supported multi format ads
- Supported [1, 1] size for video type

- contact email of the adapter’s maintainer
- [x] official adapter submission

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by haruka-yamashita2** [comment]
Hi, team (@ChrisHuie )

Excuse me for being busy, but is it likely that reviews will still take place?
How long will the review take?

Thank you.
Regards, Haruka

File: modules/yieldoneBidAdapter.js [hunk 1]
```diff
@@ -167,6 +171,107 @@ export const spec = {
...
+  }
+
+  const result = parseSizesInput(sizes).join(',');
+  return result;
+}
+
+/**
+ * @param  {Object} bidRequest -
+ * @param  {boolean} [enabledOldFormat = true] - default: `true`.
+ * @return {{w: number, h: number}} -
```
Comment (umakajan) [inline modules/yieldoneBidAdapter.js:231]: Missing `enabledFlux` here

File: modules/yieldoneBidAdapter.js [hunk 2]
```diff
@@ -167,6 +171,107 @@ export const spec = {
...
+ */
+function getBannerSizes(bidRequest, enabledOldFormat = true) {
+  let sizes = deepAccess(bidRequest, 'mediaTypes.banner.sizes');
+
+  if (enabledOldFormat) {
+    sizes = sizes || bidRequest.sizes;
+  }
+
+  const result = parseSizesInput(sizes).join(',');
+  return result;
```
Comment (umakajan) [inline modules/yieldoneBidAdapter.js:225]: Following the other comments, might be cleaner to simply `return parseSizesInput(sizes).join(',');`

**Review by umakajan — APPROVED** [review]
Minor comments, but this PR looks good

**Review by ncolletti — APPROVED** [review]
lgtm

**Comment by ncolletti** [comment]
sorry for the delay, this looks good. @haruka-yamashita2  will wait to merge until you confirm you'd like to attend to the minor feedback from @umakajan

**Comment by haruka-yamashita2** [comment]
@umakajan @ncolletti 
Thank you for your feedback.
I confirmed that, this look good, too.
Who should modify code? we should modify code and request for merge again?

Thank you.
Regards, Haruka

**Comment by umakajan** [comment]
@haruka-yamashita2 these are minor code changes, so you're welcome to change them later by letting us know or simply make the changes and add a commit to this PR. We can merge the PR after.

**Comment by haruka-yamashita2** [comment]
@umakajan 
OK, I pushed commit. Please, confirm.

Thank you.
Regards, Haruka

**Pull request merged**